intelmq/bin/intelmqdump.py

```diff
@@ -198,7 +198,7 @@ def main():
         else:
             if not answer:
                 continue
-        if answer not in available_opts:
+        if len(answer) == 0 or answer[0] not in ['s', 'r', 'e', 'a', 'd', 'q']:
```

Comment:
In case of corrupted files this does allow invalid methods, see [line 175](https://github.com/sykaeh/intelmq/blob/master/intelmq/bin/intelmqdump.py#L175)